OpenTabletDriver.Desktop/Reflection/Metadata/PluginMetadataCollection.cs

```diff
@@ -29,6 +30,8 @@ protected PluginMetadataCollection(IEnumerable<PluginMetadata> source)
 
         protected static GitHubClient GitHub { get; } = new GitHubClient(new ProductHeaderValue("OpenTabletDriver"));
 
+        public static readonly PluginMetadataCollection Empty = new PluginMetadataCollection();
```

Comment:
This should be a property, otherwise we're using the same empty instance with every empty collection.
```suggestion
        public static PluginMetadataCollection Empty => new PluginMetadataCollection();
```